scripts/reboot

```diff
@@ -51,16 +51,29 @@ function tag_images()
     fi
 }
 
+function stop_pmon_service()
+{
+     CONTAINER_STOP_RC=0
+     debug "Stopping pmon docker"
+     docker kill pmon &> /dev/null || CONTAINER_STOP_RC=$?
+     systemctl stop pmon || debug "Ignore stopping pmon error $?"
+     if [[ CONTAINER_STOP_RC -ne 0 ]]; then
+        debug "Failed killing container pmon RC $CONTAINER_STOP_RC ."
+     fi
+}
+
 function stop_sonic_services()
 {
     if [[ x"$SUBTYPE" == x"DualToR" ]]; then
         debug "DualToR detected, stopping mux container before reboot..."
         systemctl stop mux
     fi
+
     if [[ x"$ASIC_TYPE" != x"mellanox" ]]; then
         debug "Stopping syncd process..."
         docker exec -i syncd /usr/bin/syncd_request_shutdown --cold > /dev/null
         sleep 3
+        stop_pmon_service
```

Comment:
I think you should put stop_pmon_service outside the if block. Why skipping it for Mellanox platforms?